### PR TITLE
Implemented RGB565 streaming and changed max packet size

### PR DESCRIPTION
**Pull request by bartdesign**
Implemented RGB565 streaming. Changed max packet size to 1600 to prevent fragmented UDP packets.

### CODE REVIEW

File: src/ZeDMD.cpp [hunk 1]
```diff
@@ -483,7 +486,11 @@ void ZeDMD::RenderRgb565(uint16_t* pFrame)
...
 
-  if (m_usb)
+  if (m_wifi)
+  {
+    m_pZeDMDWiFi->QueueCommand(ZEDMD_COMM_COMMAND::RGB565ZonesStream, m_pPlanes, rgb565Size, width, height, 2);
+  }
+  else if (m_usb)
   {
     m_pZeDMDComm->QueueCommand(ZEDMD_COMM_COMMAND::RGB565ZonesStream, m_pPlanes, rgb565Size, width, height, 2);
   }
```
Comment (jsm174) [inline src/ZeDMD.cpp:496]: @bartdesign  - isn't this the same line? ie, if (m_wifi || m_usb) ?

Comment (bartdesign) [reply src/ZeDMD.cpp:496]: Maybe i don't understand the question, but this is the same way it has been done before in the RGB24Streaming method.

Comment (jsm174) [reply src/ZeDMD.cpp:496]: you're right. I looked too fast, and didn't see `m_pZeDMDWiFi` + `m_pZeDMDComm`. All good.

**Comment by jsm174** [comment]
I'm good with this. @mkalkbrenner ?

**Pull request merged**